### PR TITLE
Initial flatpak manifest for QiFlora

### PR DESCRIPTION
**Pull request by eyecreate**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build org.eyecreate.qiflora

File: org.eyecreate.qiflora.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+    "id": "org.eyecreate.qiflora",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "qiflora",
+    "tags": ["kde"],
```
Comment (bilelmoussaoui) [inline org.eyecreate.qiflora.json]:
```suggestion
```
Not needed.

File: org.eyecreate.qiflora.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "udev",
```
Comment (bilelmoussaoui) [inline org.eyecreate.qiflora.json]: You could remove udev if you can bump the runtime to 5.13

**Comment by nedrichards** [comment]
bot, build org.eyecreate.qiflora

**Comment by eyecreate** [comment]
bot, build org.eyecreate.qiflora

**Comment by eyecreate** [comment]
bot, build org.eyecreate.qiflora

File: org.eyecreate.qiflora.json (hunk 2 above)
Comment (eyecreate) [reply org.eyecreate.qiflora.json]: I tried using the newer runtime, but it said it couldn't find udev. I reverted it back to what works.

**Comment by barthalion** [comment]
bot, build org.eyecreate.qiflora

**Comment by barthalion** [comment]
bot, build org.eyecreate.qiflora

**Comment by barthalion** [comment]
@eyecreate Could you verify your application still works as intended? I suspect udev support in bluez is not necessary for that but I don't have hardware to test with.

**Comment by eyecreate** [comment]
I did test it and everything is still working. Thanks for doing some build testing and finding out udev could be disabled in bluez.

**Comment by barthalion** [comment]
bot, build org.eyecreate.qiflora

**Pull request closed without merging**